src/metabase/search/config.clj

```diff
@@ -85,8 +85,9 @@
   {:pinned              0
    :bookmarked          2
    :recency             1.5
-   :dashboard           1
-   :model               0.5
+   :user-recency        1.5
+   :dashboard           0.5
```

Comment:
Rationale for weakening this - only cards get a score for it, so its artificially boosting them too much.